src/gluonts/mx/distribution/categorical.py

```diff
@@ -121,7 +121,7 @@ def domain_map(self, F, probs):
     def distribution(
         self, distr_args, loc=None, scale=None, **kwargs
     ) -> Distribution:
-        distr = Categorical(distr_args)
+        distr = Categorical(*distr_args)
```

Comment:
This could go in a separate PR, along with some tests

Reply:
As discussed offline, this was a mistake and I removed it again.